modules/core/src/main/java/org/apache/ignite/internal/processors/task/monitor/ComputeTaskStatus.java

```diff
@@ -298,7 +299,7 @@ public static ComputeTaskStatus onFinishTask(GridTaskSessionImpl sessionImp, @Nu
             null,
             null,
             0L,
-            0L,
+            U.currentTimeMillis(),
```

Comment:
Why not "sessionImpl.getEndTime()"?

Reply:
This is the estimated time of the end of the task, so for example to cancel the task by timeout.

Reply:
Ok, I thought that this is the actual end time... I think you should ask CC team for what they want from this value, my understanding might be wrong